README.md

```diff
@@ -114,7 +115,7 @@ basic-http-server .
 
 ## android
 
-Recommended way to build for android is using Docker.   
+Recommended way to build for android is using Docker.
```

Comment:
Trailing whitespaces in markdown make a newline. 
Without those whitespaces, formatting in README on GH is slightly corrupted :)

Reply:
`<br/>` should work as well, and it doesn't get removed by the tools that clean trailing spaces up automatically

Reply:
>  <br/> should work as well, and it doesn't get removed by the tools that clean trailing spaces up automatically

Good point, <br/> may be easier to maintain.

Reply:
@mvlabat thanks! Fix it.